src/dotnet-interactive-vscode/package.json

```diff
@@ -1,7 +1,8 @@
 {
   "name": "dotnet-interactive-vscode",
   "displayName": ".NET Interactive Notebooks",
-  "description": ".NET Interactive Notebooks for VS Code",
+  "//description": "During build, the description gets the git SHA appended to it.",
```

Comment:
Did you mean to leave this in?

Reply:
Yes, because if you run the script locally you'll have edits to this file that shouldn't get checked in; this is a sad attempt at documenting this.